### PR TITLE
Sync galley templates with master

### PR DESCRIPTION
**Pull request by howardjohn**
Signed-off-by: John Howard <howardjohn@google.com>

### CODE REVIEW

File: istio-control/istio-config/templates/_podDisruptionBudget.tpl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{- define "podDisruptionBudget.spec" }}
```
Comment (costinm) [inline istio-control/istio-config/templates/_podDisruptionBudget.tpl]: I'm  trying to minimize templating - you can just inline it, it's not used in any other place.

File: istio-control/istio-config/templates/deployment.yaml [hunk 2]
```diff
@@ -20,20 +24,24 @@ spec:
...
+{{- if contains "/" .Values.galley.image }}
+          image: "{{ .Values.galley.image }}"
+{{- else }}
+          image: "{{ .Values.global.hub }}/{{ .Values.galley.image | default "galley" }}:{{ .Values.global.tag }}"
+{{- end }}
           imagePullPolicy: {{ .Values.global.imagePullPolicy | default "Always" }}
           ports:
           - containerPort: 443
-          - containerPort: {{ .Values.global.monitoringPort | default "15014" }}
+          - containerPort: {{ .Values.global.monitoringPort }}
```
Comment (costinm) [inline istio-control/istio-config/templates/deployment.yaml]: Can we leave it ? monitoringPort is another setting we may deprecate, there is no reason to customize it or plan to test such customization.

File: istio-control/istio-config/templates/service.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ spec:
   ports:
   - port: 443
     name: https-validation
-  - port: 15014
+  - port:  {{ .Values.global.monitoringPort }}
```
Comment (costinm) [inline istio-control/istio-config/templates/service.yaml]: Same. If we can hardcode 9901 and 443, there is no reason to customize monitoringPort.

File: istio-control/istio-config/templates/deployment.yaml [hunk 4]
```diff
@@ -20,20 +24,24 @@ spec:
...
+{{- else }}
+          image: "{{ .Values.global.hub }}/{{ .Values.galley.image | default "galley" }}:{{ .Values.global.tag }}"
+{{- end }}
           imagePullPolicy: {{ .Values.global.imagePullPolicy | default "Always" }}
           ports:
           - containerPort: 443
-          - containerPort: {{ .Values.global.monitoringPort | default "15014" }}
+          - containerPort: 15014
           - containerPort: 15019
+          - containerPort: 9901
```
Comment (costinm) [inline istio-control/istio-config/templates/deployment.yaml]: This moved to the sidecar.

Comment (howardjohn) [reply istio-control/istio-config/templates/deployment.yaml]: So we don't need this port now?

**Pull request merged**